components/ShakeAPI.js

```diff
@@ -0,0 +1,112 @@
+import React, { useState, useEffect } from 'react'
+import { View, Text, TouchableOpacity } from 'react-native'
+import styled from 'styled-components/native'
+import { Accelerometer } from 'expo-sensors'
+
+const Container = styled.View`  
+  border: 2px solid #655D8A;
+  border-radius: 6px;
+`
+
+const QuoteContent = styled.Text`
+  font-size: 20px; 
+  background-color: #655D8A;
+  color: white;
+  padding: 40px;
+  line-height: 30px;
+`
+
+const QuoteAuthor = styled.Text`
+  font-size: 16px;
+  font-style: italic;
+  text-align: right;
+  padding-right: 12px;
+  padding-bottom: 6px;
+  background-color: #655D8A;
+  color: #FFEEEE;
+`
+
+const Button = styled.TouchableOpacity`
+  width: 100%;
+  background-color: #D885A3;
+  padding: 20px;
+  border-radius: 6px;
+  margin-top: 20px;
+`
+
+const ButtonText = styled.Text`
+  font-size: 20px;
+  color: white;
+`
+
+const ShakeAPI = () => {
```

Comment:
I like that you put the fetch of the API in here, good structure and easy to follow